config/development.json

```diff
@@ -1,9 +1,13 @@
 {
-  "INBOUND": {
-    "PORT": 4005,
-    "HOST": "0.0.0.0",
-    "PISP_TRANSACTION_MODE": false,
-    "TLS": {
+  "control": {
+    "mgmtAPIWsUrl": "127.0.0.1",
+    "mgmtAPIWsPort": 4010
+  },
+  "inbound": {
```

Comment:
Would UPPER-CASE to CAME-CASE be a breaking change here?

Reply:
Unsure. So...I'm going to make it one.